pkg/infra/spot.go

```
@@ -238,6 +241,7 @@ var _ = Describe("[sig-cluster-lifecycle] Machine API Running on Spot", framewor
 		})
 
 		By("should terminate a Machine if a termination event is observed", func() {
+
```

golangci-lint (2.12.2)

[error] unnecessary whitespace (leading-whitespace)

(wsl_v5)